src/XRP/default-xrp-client.ts

```diff
@@ -484,6 +485,43 @@ export default class DefaultXrpClient implements XrpClientDecorator {
     return await this.getTransactionResult(transactionHash)
   }
 
+  /**
+   * Disables DepositPreauth and unauthorizes an XRPL account to send to this XRPL account.
+   *
+   * @set https://xrpl.org/depositpreauth.html
+   *
+   * @param xAddressToUnauthorize The X-Address of the sender to unauthorize.
+   * @param wallet The wallet associated with the XRPL account disabling DepositPreauth and that will sign the request.
```

Comment:
```suggestion
   * @param wallet The wallet associated with the XRPL account revoking a deposit preauthorization, and that will sign the request.
```

Reply:
this language just seems to make it more clear that there is a revocation happening toward a different address as opposed to changing some setting on the wallet account

Reply:
cool, i like this - i've also updated the doc on authorize